src/jobflow/managers/local.py

```diff
@@ -165,4 +165,4 @@ def _run(root_flow):
     if ensure_success and not finished_successfully:
         raise RuntimeError("Flow did not finish running successfully")
 
-    return dict(responses)
+    return responses
```

Comment:
This is actually purposeful to convert from defaultdict to dict. Purely a style thing but I think this should stay as is?

Reply:
`defaultdict` subclasses `dict` so for the purpose of `mypy`, there's no need to convert. At run time `defaultdict` offers all the methods of `dict` too so no potential for problems there either (afaik). But happy to revert if you prefer.

Reply:
Yeah, its not for the API, functionality, or mypy. Just purely stylistic when working with jobflow in a jupyter notebook for example. Thanks for reverting.